path_roles.go

```diff
@@ -206,6 +212,9 @@ func (b *backend) pathRolesWrite(ctx context.Context, req *logical.Request, d *f
 	if tokenTTLRaw, ok := d.GetOk("token_default_ttl"); ok {
 		entry.TokenDefaultTTL = time.Duration(tokenTTLRaw.(int)) * time.Second
 	}
+	if tokenAudiencesRaw, ok := d.GetOk("token_default_audiences"); ok {
+		entry.TokenDefaultAudiences = strutil.RemoveDuplicates(tokenAudiencesRaw.([]string), true)
```

Comment:
It looks like audiences can be mixed case, so we might want to use

```suggestion
		entry.TokenDefaultAudiences = strutil.RemoveDuplicates(tokenAudiencesRaw.([]string), false)
```